test/main.cpp

```diff
@@ -4,10 +4,30 @@
 #include <catch2/catch_session.hpp>
 #include <locale>
 
+#include <QSurfaceFormat>
+
 int main(int argc, char *argv[]) {
   Catch::StringMaker<double>::precision = 25;
   Catch::StringMaker<float>::precision = 25;
 
+  QSurfaceFormat format;
+  // format.setProfile(QSurfaceFormat::CoreProfile);
```

Comment:
delete commented code?

Reply:
This one was left intentionally as a reminder for whenever we can test using a Mac machine.

Reply:
Ah ok, then could you add that info as a comment here?